src/platforms/react-native/performance/included-instrumentation.mdx

```diff
@@ -134,22 +134,34 @@ class App extends React.Component {
   // Create a ref for the navigation container
   navigation = React.createRef();
 
-  componentDidMount() {
-    // Register the navigation container with the instrumentation
-    routingInstrumentation.registerNavigationContainer(navigation);
-  }
-
   render() {
     return (
       // Connect the ref to the navigation container
-      <NavigationContainer ref={this.navigation}>
+      <NavigationContainer
+        ref={this.navigation}
+        onReady={() => {
+          // Register the navigation container with the instrumentation
+          routingInstrumentation.registerNavigationContainer(navigation);
+        }}>
         ...
       </NavigationContainer>
     )
   }
 }
 ```
 
+#### Configuration
```

Comment:
would it make sense to move this as part of the content discussed on line 15 (Configuration Options)?

Reply:
Hmm I don't think so, as these configuration options are unique to the React Navigation Instrumentation (future instrumentation might not have them, also the behavior on V4 and V5 differ slightly) and the configuration section above is for `ReactNativeTracing`